.github/workflows/docker.yml

```diff
@@ -57,15 +59,25 @@ jobs:
           - sytest_image_tag: focal
```

Comment:
the old `sytest_image_tag` variables seem to be unused now. Could you get rid of them?

Reply:
I believe they are still being used on line 54?:
```
  name: "Build sytest-${{ matrix.dockerfile }}:${{ matrix.sytest_image_tag }}"
```